BFH_USZ_PICC/BFH_USZ_PICC/Views/SettingsPage.xaml.cs

```diff
@@ -50,7 +50,7 @@ void AddNotificationButtonClicked(object o, EventArgs e)
 
         void PersonalMasterDataClicked(object o, EventArgs e)
         {
-            Navigation.PushAsync(new BasePage(typeof(UserMasterDataPage), new List<object> { UserMasterDataPageDisplayMode.View }));
+            Navigation.PushAsync(new BasePage(typeof(UserMasterDataPage), new List<object> { false }));
```

Comment:
false = UserMasterDetail page won't be opened in "enable" mode (user needs to press the "Edit" button before he/she can edit the personal data)